backend/__init__.py

```diff
@@ -27,16 +28,34 @@
 from backend.config import EnvironmentConfig
 
 
+# Storing error messages in a json file so that it is loaded only once at startup (Used in exceptions.py)
+ERROR_MESSAGES = json.load(open("backend/error_messages.json"))
```

Comment:
This is not the best way to handle file open since we usually forget to close. Please use the better approach to automatically close open file.

```
with open("backend/error_messages.json") as jsonfile:
  ERROR_MESSAGES = json.load(jsonfile)
```

Reply:
Additionally, in a later PR. Please also consider using module.path style something for the JSON file location.